recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -20,41 +22,42 @@ WPEFRAMEWORK_SYSTEM_PREFIX ??= "OE"
 # Configure exceptions handling (ON/OFF)
 WPEFRAMEWORK_EXCEPTIONS_ENABLE ??= "OFF"
 
-PACKAGECONFIG ??= " release \
+PACKAGECONFIG ??= "\
     ${@bb.utils.contains('MACHINE_FEATURES', 'bluetooth', 'bluetooth', '', d)} \
-    websource webkitbrowser"
+    webserver webkitbrowser \
```

Comment:
I would add webserver and webkitbrowser dynamically when the plugin is available. Additionally I would change the name, as we are enabling here the auto resume mechanism not the plugins itself

Reply:
renamed the webserver and webkitbrowser, but still kept as static since not able to set dynamic dependency using PACKAGECONFIG